### ISSUE TITLE
Remove author as this info is not important

### ISSUE THREAD

**Issue by simonschaufi** [dantleech/what-changed#16]
Removing the author will give more space for the actual commit message title

**Comment by dantleech** [comment]
It's important to me -- it would be better to make it configurable (we already have the [options](https://github.com/dantleech/what-changed/blob/master/lib/Model/ReportOptions.php) but they are only used by the console command, not the plugin).

I think we'd need to add some `config` to `composer.json`

**Comment by simonschaufi** [comment]
I saw in the readme the 2 parameters but how do I configure the plugin system wide as I have it installed in global scope?

**Comment by dantleech** [comment]
I don't think this is possible currently. The solution would probably be to have:

```
"extra": {
    "what_changed": {
         // config here
     }
}
```

in `composer.json` (which is in `~/.composer/comoser.json`) globally.

(note that currently the plugin doesn't use github auth (#8) - so you will likely hit the Github API limit if you do lots of composer updates)
